### DISCUSSION TITLE
Scripting language: Syntax for Sonoff Smart Plug energy readings

### DISCUSSION THREAD

**Discussion by Kugeleis** [arendst/Tasmota#12788] [Q&A] [answered]
While with rules I can write
`on ENERGY#Power DO Var1 %value% endon `
using the scripting language I have no success with
`StatusSNS#ENERGY#Total`
despite the command status 8 spits out
`{"StatusSNS":{"Time":"2021-07-30T08:46:41","ENERGY":{"TotalStartTime":"2020-05-16T18:41:30","Total":209.215,"Yesterday":0.000,"Today":0.000,"Power":0,"ApparentPower":0,"ReactivePower":0,"Factor":0.00,"Voltage":0,"Current":0.000}}}`
What is the correct scripting language syntax to access the energy readings? Is there documentation on it?

**Comment by barbudor** [comment, marked as answer]
If you mean in your `>T` section, it should be like:
```
>T
pow=ENERGY#Power
tot=ENERGY#Total
```
To process the standard telemetry report 

And by the way there was an example in the doc at https://tasmota.github.io/docs/Scripting-Language/#send-e-mail

**Comment by Kugeleis** [comment]
Ah, ok, this works. My error was to expect it in the >S section.
